### PR TITLE
smol addition of squaresupply

### PR DESCRIPTION
**Pull request by tanishq-soni**
@zfogg

### CODE REVIEW

File: src/lib/emojiKeywords.js [hunk 1]
```diff
@@ -175,5 +175,6 @@ export default {
   "cold": "snowflake",
   "tailwind": "tailwind",
   "tailwindcss": "tailwind",
-  "c": "c"
+  "c": "c",
+  "nft": "squaresupply"
```
Comment (sampoder) [inline src/lib/emojiKeywords.js:179]: Personally, I'd prefer to use general emojis for general topics or a least NFTs with a generalised name so we get `/r/nft` etc.

**Review by sampoder — CHANGES_REQUESTED** [review]
view above ^^

File: src/lib/emojiKeywords.js (hunk 1 above)
Comment (tanishq-soni) [reply src/lib/emojiKeywords.js:179]: squaresupply is an NFT project by zfogg.

Comment (zfogg) [reply src/lib/emojiKeywords.js:179]: i actually agree with @sampoder here, @tanishq-soni. make "square supply", "squaresupply.xyz", and "squaresupply" work for the `:squaresupply:` emoji.. the word "NFT" is too broad to apply to the `:squaresupply:` emoji lol

Comment (tanishq-soni) [reply src/lib/emojiKeywords.js]: okay 👍🏻

**Review by sampoder — APPROVED** [review]

**Pull request merged**